packages/core/lib/core/format-utils.js

```diff
@@ -22,13 +22,12 @@ export const DOCK_TOKEN_UNIT = 1000000;
  * @returns string
  */
 export function formatCurrency(value, currency = 'USD'): string {
-  assert(!!value, 'value is required');
   assert(
     typeof value === 'number' || typeof value === 'bigint',
     'value must be a number or bigint',
   );
 
-  var formatter = new Intl.NumberFormat('en-US', {
```

Comment:
Should we get the locale from some state maybe? Wondering how we can switch this to use the phone defaults later on. Perhaps we should create some Intl manager file, or something to store the current locale selection

Reply:
How about we pass the locale via the function parameter params? We can default it to 'en-US'